TLM/TLM/UI/SubTools/LaneArrows/LaneArrowTool.cs

```diff
@@ -215,6 +217,9 @@ private enum Trigger {
                 buttonLeft.ToggleFlag = API.Traffic.Enums.LaneArrows.Left;
                 buttonLeft.UpdateButtonSkinAndTooltip();
                 buttonLeft.ParentTool = this; // to access error reporting function on click
+                bool leftAllowed =  (availableArrows & LaneArrows.Left) != 0;
+                buttonLeft.isEnabled = leftAllowed;
+                buttonLeft.tooltip = !leftAllowed ? T("LaneArrow: Direction not available") : string.Empty;
```

Comment:
Translations, have they been updated?
![image](https://user-images.githubusercontent.com/288724/223881346-ada86c9f-b159-4991-abf9-2c5240250ad9.png)

Reply:
true, need to be updated/string needs to be added 😉

Reply:
I'll create new PR with translations